app/services/GitRepositoryService.scala

```diff
@@ -4,22 +4,25 @@ import java.io.File
 import java.net.URL
 
 import javax.inject.{Inject, Singleton}
+import model.CustomExecutionContext.executionContextExecutor
 import org.eclipse.jgit.api.Git
 import org.eclipse.jgit.api.errors.RefNotAdvertisedException
 import org.eclipse.jgit.internal.storage.file.FileRepository
 import org.eclipse.jgit.transport.UsernamePasswordCredentialsProvider
 import play.api.ConfigLoader.stringLoader
 import play.api.libs.json.{JsValue, Json}
-import play.api.{Configuration, Logger}
+import play.api.{ConfigLoader, Configuration, Logger}
 
-import scala.concurrent.ExecutionContext.Implicits.global
-import scala.concurrent.duration.Duration
 import scala.concurrent.{Await, Future}
 import scala.io.Source
+import scala.language.postfixOps
+
 
 @Singleton
 class GitRepositoryService @Inject()(configuration: Configuration) {
 
+  private val projectUrl = """https?:\/\/[^ \/]+\/(.*)""".r
```

Comment:
To match the modification in `MavenVersionFetcher`, `"` can probably be used instead of `"""`.

Reply:
le `"""` m'évite de faire des `\\`